packages/asset-service/src/generateAssetData/Utils.test.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseAsset } from '@shapeshiftoss/types'
+
+import { BTCMockedAsset, ETHMockedAsset } from '../service/AssetServiceTestData'
+import blacklist from './blacklist.json'
+import { filterBlacklistedAssets } from './utils'
+
+const blacklistedAssets: string[] = blacklist
+
+jest.mock('./blacklist.json', () => [ETHMockedAsset.tokens![0].caip19, BTCMockedAsset.caip19], {
```

Comment:
@gomesalexandre @0xdef1cafe I'm really not sure about `ETHMockedAsset.tokens![0].caip19`

It's some mocked data, we know that `tokens` property exist, but TS throw an error

If it doesn't exist, jest throw an error, so I guess it should be fine, but if you have a better way...

Reply:
fixed using as const in mocks